R/call.R

```diff
@@ -22,7 +22,7 @@
 #' @param .fn Function to call. Must be a callable object: a string,
 #'   symbol, call, or a function.
 #' @param ... Arguments to the call either in or out of a list. These dots
-#'   support [tidy dots][tidy-dots] features. Empty arguments are preserved.
+#'   are [dynamic][dyn-dots] and preserve empty arguments.
```

Comment:
Maybe a little more context: These are [dynamic dots][dyn-dots] which support `!!!` and `:=`, and preserve empty arguments.

Reply:
Would this be just for rlang? I think in dplyr or tidyverse packages it might be best to leave off all the scary looking terms or names.